### PR TITLE
docs: restructured support matrix

### PR DESCRIPTION
**Pull request by nyrahul**
**Purpose of PR?**:

Restructures kubearmor support matrix.

**Does this PR introduce a breaking change?**
No

**Additional information for reviewer?** :
Check if the links in the PR are not broken.

**Checklist:**
- [x] This change requires a documentation update
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`

### CODE REVIEW

File: getting-started/support_matrix.md [hunk 1]
```diff
@@ -1,60 +1,83 @@
...
 
-It would be very much appreciated if you can test kubearmor on a platform not listed above and if you have access to. Once tested you can update this document and raise a PR, if possible.
-
-### What local K8s platforms are supported?
-
-[Minikube](../contribution/minikube), [K3s](../deployments/k3s) and [Microk8s](../contribution/microk8s) platforms are currently supported.
-
-### Why KubeArmor does not work on kind
-
-KubeArmor does not support Kubernetes in Docker.
```
Comment (daemon1024) [inline getting-started/support_matrix.md:59]: Should we mention about supporting only observability rn in KIND?

**Review by Ankurk99 — APPROVED** [review]
LGTM!

File: getting-started/support_matrix.md [hunk 2]
```diff
@@ -1,60 +1,83 @@
...
+| AWS        | [Graviton] | Ubuntu | ARM | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | AppArmor |
+| AWS        | [Graviton] | Amazon Linux 2 | ARM | :heavy_check_mark: | :heavy_check_mark: | :x: | :heavy_check_mark: | SELinux |
+| RedHat     | [OpenShift] | [RHEL] <=8.4 | x86_64 | :heavy_check_mark: | :heavy_check_mark: | :x:  | :heavy_check_mark: | SELinux |
+| RedHat     | [OpenShift] | [RHEL] >=8.5 | x86_64 | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | [BPFLSM] |
+| Rancher    | [RKE] | [SUSE] | x86_64 | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | [BPFLSM], AppArmor |
+| Rancher    | [K3S] | [Distros] | x86_64 | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | [BPFLSM], AppArmor |
+| VMWare     | [Tanzu] | TBD | x86_64 | :construction: | :construction: | :construction: | :construction: | :construction: | [1064] |
+| Oracle     | [Ampere] | [UEK] | ARM | :construction: | :construction: | :construction: | :construction: | :construction: | [1084] |
```
Comment (rksharma95) [inline getting-started/support_matrix.md]: we can conclude support on Ampere, currently only Observability supported.

File: getting-started/support_matrix.md (hunk 1 above)
Comment (nyrahul) [reply getting-started/support_matrix.md:59]: Do not want to project local environments that we do not support fully?

File: getting-started/support_matrix.md (hunk 2 above)
Comment (nyrahul) [reply getting-started/support_matrix.md]: Done. updated the doc.

**Review by rksharma95 — APPROVED** [review]
LGTM :+1:

**Pull request merged**